packages/node-client/src/NodeClient.ts

```diff
@@ -126,7 +126,7 @@ class NodeClient implements INodeClient {
    * @description this function will return the signature for your domain
    * @returns
    */
-  async whitelistUrl(origin: string): Promise<WhiteListSignatureResponse> {
+  async whiteListUrl(origin: string): Promise<WhiteListSignatureResponse> {
```

Comment:
This change is not needed. In general English usage, "whitelist" is one word.

whiteListUrl might imply that you're listing "white" objects, which doesn't seem to be the intention here

Reply:
reverted